src/index/TextIndexLiteralConfiguration.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2025, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Felix Meisen (fesemeisen@outlook.de)
+
+#ifndef QLEVER_SRC_INDEX_TEXTINDEXLITERALCONFIGURATION_H
+#define QLEVER_SRC_INDEX_TEXTINDEXLITERALCONFIGURATION_H
+
+#include "index/TextIndexLiteralFilter.h"
+
+struct TextIndexLiteralConfiguration {
+  std::string regex_;
```

Comment:
```suggestion
  std::string predicateRegex_;
```

and also add some documentation to the semantics of these members.